### PR TITLE
Feature/801 deployment

### PR DESCRIPTION
**Pull request by julius-heitkoetter**
Created scripts for the 8.01 deployment. Basically just copied all the prod stuff over, replaced `prod` with `prod-801` and then double checked that all the replacements were correct.

### CODE REVIEW

**Comment by julius-heitkoetter** [comment]
Do not merge until #77 is resolved

File: .github/workflows/prod-801-ci-cd.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+      # copy repository to machine
+      - name: Copy Repository
+        run: |
+          rsync -e ssh -r ${{ github.workspace}}/* --exclude .git/ --delete submit-t3desk:~/A2rchi-prod-801/
+
+      # run deploy script
+      - name: Run Deploy Script
+        run: |
+          ssh submit-t3desk 'bash -s' < ${{ github.workspace }}/deploy/prod/prod-801-install.sh
```
Comment (mdr223) [inline .github/workflows/prod-801-ci-cd.yaml:59]: Need to change `deploy/prod/` --> `deploy/prod-801/`

File: config/prod-801-config.yaml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    static_folder: "/root/A2rchi/A2rchi/interfaces/chat_app/static"
+  uploader_app:
+    PORT: 5001
+    HOST: "0.0.0.0" # either "0.0.0.0" (for public) or "127.0.0.1" (for internal)
+    template_folder: "/root/A2rchi/A2rchi/interfaces/uploader_app/templates"
+
+chains:
+  input_lists:
+    - empty.list
+    - submit.list
```
Comment (mdr223) [inline config/prod-801-config.yaml:27]: Does this class use `SubMIT` or is it intentional to leave this in?

**Review by mdr223 — CHANGES_REQUESTED** [review]
Left two comments, one of which requires a change for correctness. Otherwise it all looks good!

File: config/prod-801-config.yaml (hunk 2 above)
Comment (mdr223) [reply config/prod-801-config.yaml]: Removing for now, can always add back.

**Review by mdr223 — APPROVED** [review]
Made minor tweaks; updated docker-compose to mount 801 content which is now on t3desk

**Pull request merged**